src/main/webapp/script.js

```diff
@@ -47,21 +47,25 @@ function cleanseUrl(url) {
 /**
  * Fetches data and adds to html
  */
-function getChart() {
-  $('form').submit(function() {
+async function getChart() {
+  $('form').submit(async function() {
+    commentStats = await getYouTubeComments();
+    averageScore = commentStats.averageScore;
+    aggregatedValues = commentStats.aggregateValues; 
+
     const CommentSentimentTable = new google.visualization.DataTable();
     CommentSentimentTable.addColumn('string', 'SentimentRange');
     CommentSentimentTable.addColumn('number', 'CommentCount');
 
-    for (currentLabel = LOWEST_SCORE; currentLabel < HIGHEST_SCORE; currentLabel += INTERVAL) {
-      // TODO: Replace abritrary value 6 with correct aggregation value  
+    currentLabel = LOWEST_SCORE;
+    Object.keys(aggregatedValues).sort((a,b) => a-b).forEach(function(key) {
```

Comment:
nit: spacing: "(a, b) => a - b"

Reply:
Can you write a comment here to explain what the keys are and how they are sorted?